src/data/github-projects.json

```diff
@@ -432,6 +432,30 @@
             "repoName": "hiroppy/fusuma",
             "logo": "fusuma.svg"
         },
+        {
+            "title": "Terasology",
+            "description": "Terasology - open source voxel world",
+            "language": "Java",
+            "tags": [
+                {
+                    "href": "https://github.com/MovingBlocks/Terasology/releases/latest",
+                    "src": "https://img.shields.io/github/downloads/MovingBlocks/Terasology/total.svg",
+                    "alt": "Downloads"
+                },
+                {
+                    "href": "https://www.bountysource.com/teams/MovingBlocks",
+                    "src": "https://img.shields.io/bountysource/team/MovingBlocks/activity.svg",
+                    "alt": "Bounties"
+                },
```

Comment:
It says `bounties | inaccessible`, so I'll remove this badge for now.

<img width="1118" alt="Screenshot 2020-04-07 at 16 27 30" src="https://user-images.githubusercontent.com/599268/78681201-b4d9a980-78ec-11ea-87fc-b7badeda2b7f.png">

```suggestion
```